courses/free-courses-en.md

```diff
@@ -1245,7 +1250,9 @@
 * [Intro to Rust](https://www.youtube.com/playlist?list=PLJbE2Yu2zumDF6BX6_RdPisRVHgzV02NW) - Tensor Programming
 * [Learn Rust from scratch](https://www.educative.io/courses/learn-rust-from-scratch) - Educative.io
 * [Rust Basics](https://www.youtube.com/playlist?list=PLlcnQQJK8SUjApd95LIcd3K9XXmE-IeCS) - Engineer Man
+* [Rust Books](https://github.com/sger/RustBooks) - Books
 * [Rust Crash Course](https://www.youtube.com/watch?v=zF34dRivLOw) - Traversy Media
+* [Rust Learnings](https://github.com/ctjhoa/rust-learning) - a list of awesome rust learning material
```

Comment:
Add each resource link directly

Reply:
Rust learning is seperate repo for listing out books, videos and various resources on learning rust.
Its impossible to update this repo with that all the time. and as this repo doesnt have a seperate list for repos or other resources for this to fit in i'll just remove this line from here.